deploy-manage/autoscaling.md

```diff
@@ -4,67 +4,208 @@ mapped_urls:
   - https://www.elastic.co/guide/en/cloud/current/ec-autoscaling.html
   - https://www.elastic.co/guide/en/cloud-enterprise/current/ece-autoscaling.html
   - https://www.elastic.co/guide/en/elasticsearch/reference/current/xpack-autoscaling.html
+applies_to:
+  deployment:
+    ece: ga
+    ess: ga
+    eck: ga
 ---
 
 # Autoscaling
 
-% What needs to be done: Refine
+::::{note} 
+{{cloud-only}}
+::::
```

Comment:
This note doesn't render properly. you need to edit the contents of the `{{cloud-only}}` var so it uses the markdown link formatting.

consider instead: 

```
::::{admonition} Indirect use only 
This feature is designed for indirect use by {{ech}}, {{ece}}, and {{eck}}. Direct use is not supported.
::::
```